packages/devtools/client/src/styles/fonts.scss

```diff
@@ -0,0 +1,11 @@
+/*
+ * This subset is modified from the Noto Sans font,
+ * which is distributed under the Open Font License.
+ * You can learn more about it at the following link.
+ * https://openfontlicense.org/
+ * https://fonts.google.com/noto/specimen/Noto+Sans
+ */
+@font-face {
+  font-family: 'NotoSans';
+  src: url('@/assets/NotoSans-latin-subset.ttf');
```

Comment:
woff2 is smaller

Reply:
can we use woff2

Reply:
Yep, I converted it to woff2 format, resulting in a significantly smaller size (70KB -> 38 KB).

Reply:
👍